compiler/native/script.go

```diff
@@ -98,16 +98,21 @@ func generateScriptPosix(commands []string) string {
 // setupScript is a helper script this is added to the build to ensure
 // a minimum set of environment variables are set correctly.
 const setupScript = `
-cat <<EOF > $HOME/.netrc
-machine $VELA_NETRC_MACHINE
-login $VELA_NETRC_USERNAME
-password $VELA_NETRC_PASSWORD
+set +e
```

Comment:
"+e" is essentially default behavior, you are unsetting it here - is that the intent?

Reply:
i failed to realize we `set -e`. so the comments related to (un)setting this flag are to be ignored :disappear: